.github/workflows/main.yml

```diff
@@ -58,3 +78,20 @@ jobs:
         with:
           name: juju-crashdump
           path: juju-crashdump-*.tar.xz
+
+  charmhub-upload:
+    runs-on: ubuntu-20.04
+    name: Charmhub upload
+    needs:
+      - unit-tests-with-coverage
+      - integration-test
+    if: github.ref_name == '1.8'
+    steps:
+      - uses: actions/checkout@v3
+      - name: Upload charm to charmhub
+        uses: canonical/charming-actions/upload-charm@2.2.2
```

Comment:
Version `2.2.5` is out, any reason why we're not freezing to the latest?

Reply:
Updated.